base/irrationals.jl

```diff
@@ -185,3 +185,8 @@ function alignment(io::IO, x::AbstractIrrational)
     m === nothing ? (length(sprint(show, x, context=ctx, sizehint=0)), 0) :
     (length(m.captures[1]), length(m.captures[2]))
 end
+
+# inv
+inv(x::AbstractIrrational) = 1/x
+@test inv(π) ≈ 0.3183098861837907
+@inferred inv(π)
```

Comment:
The test should go to `test/numbers.jl` and be `@test @inferred(inv(π)) ≈ 0.3183098861837907`

Reply:
Thanks, I moved and consolidated the tests.